packages/contentstack-config/src/commands/config/get/rate-limit.ts

```diff
@@ -0,0 +1,52 @@
+import { cliux, configHandler, isAuthenticated } from '@contentstack/cli-utilities';
+import { Command } from '@contentstack/cli-command';
+import { handleErrorMsg } from '../../../utils/interactive';
+import { RateLimitConfig } from '../../../interfaces';
+
+export default class RateLimitGetCommand extends Command {
```

Comment:
@harshithad0703 , description is missing

Reply:
added